package.json

```diff
@@ -12,14 +12,22 @@
     "precommit": "npm run lint"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.2",
+    "@edx/eslint-config": "^2.0.0",
     "cypress": "^5.1.0",
     "cypress-failed-log": "^2.5.0",
     "cypress-file-upload": "^4.1.1",
-    "eslint": "^6.1.0",
-    "eslint-config-edx": "^4.0.4",
+    "eslint": "^8.10.0",
+    "eslint-config-airbnb": "^19.0.4",
     "eslint-plugin-cypress": "^2.6.1",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.2",
+    "eslint-plugin-react-hooks": "^4.3.0",
     "googleapis": "42.0.0",
     "uuid": "^3.3.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.5",
+    "@babel/eslint-parser": "^7.17.0"
```

Comment:
Please add the reference or reason for this change

Reply:
These packages were installed due to `@edx/eslint-config`. `eslint-config-edx` is deprecated and doesn't support node16 so we have to replace that with newer package `@edx/eslint-config`

Reply:
@babel/core and @babel/eslint-parser are also part of this?

Reply:
yeah these are also installed with it because the parser `babel-eslint` is also deprecated and replaced with `@babel/eslint-parser` and `@babel/core` is also its dependency.